lib/native/http.ts

```diff
@@ -87,4 +88,103 @@ export class HttpClientConnection extends HttpConnection {
         protected tls_ctx?: ClientTlsContext) {
         super(native_handle);
     }
+
+    make_request(request: HttpRequest, on_response: StreamResponseCallback, on_body: StreamBodyCallback) {
+        let stream: HttpClientStream;
+        const on_response_impl = (status_code: Number, headers: string[][]) => {
+            stream._on_response(status_code, headers);
+        }
+
+        const on_body_impl = (data: ArrayBuffer) => {
+            stream._on_body(data);
+        }
+        
+        const on_complete_impl = (error_code: Number) => {
+            stream._on_complete(error_code);
+        }
+        const native_handle = crt_native.http_stream_new(
+            this.native_handle(),
+            request.method,
+            request.path,
```

Comment:
I guess that would matter for non-native signers. Do we intend to have those? I guess since we're binding the signing interface...I can do it later, it's not a huge deal to do.